crates/bevy_ui/src/render/mod.rs

```diff
@@ -844,43 +875,52 @@ pub fn extract_uinode_text(
         transform.translation = transform.translation.round();
         transform.translation *= inverse_scale_factor;
 
-        let mut color = LinearRgba::WHITE;
-        let mut current_section = usize::MAX;
-        for PositionedGlyph {
-            position,
-            atlas_info,
-            section_index,
-            ..
-        } in &text_layout_info.glyphs
+        for (
+            i,
+            PositionedGlyph {
+                position,
+                atlas_info,
+                section_index,
+                ..
+            },
+        ) in text_layout_info.glyphs.iter().enumerate()
```

Comment:
nit: this is a bit hard to read, moving the `PositionedGlyph` destructuring into the for loop body could make it easier

Reply:
I don't know. I prefer the more concise deconstruction,it uses less vertical space and the inner code block isn't that large . It's not that important though if consensus favours the other way I don't mind changing it.